config/karma.config.js

```diff
@@ -42,13 +38,25 @@ module.exports = function (config) {
     colors: true,
     logLevel: config.LOG_INFO,
     autoWatch: false,
-    browsers: ['Chrome'],
-	customLaunchers: {
-		ChromeTravisCi: {
-			base: 'Chrome',
-			flags: ['--no-sandbox']
-		}
-	},
+    browsers: ['PhantomJS', 'PhantomJS_custom'],
+    customLaunchers: {
+      'PhantomJS_custom': {
+        base: 'PhantomJS',
+        options: {
+          windowName: 'alm-window',
+          settings: {
+            webSecurityEnabled: false
+          },
+        },
+        flags: ['--load-images=true'],
+        debug: true
+      }
+    },
+    phantomjsLauncher: {
```

Comment:
This commit message is the opposite of the change. LOL

Please reword the message.